setuper/README.md

```diff
@@ -76,4 +76,4 @@ Once the script has run, you can log in to your server using the account name as
 
     You will need to add param `--additionnal_projects`:
 
-        python3 setuper.py --additionnal_projects
\ No newline at end of file
+        python3 setuper.py -a
```

Comment:
I tend to think that `--additionnal_projects` is more explicit than `-a`.

What motivated this change?